requirements-test.txt

```diff
@@ -1,9 +1,10 @@
 img2dataset
 black==22.3.0
 mypy==0.942
+open-clip-torch==0.2.1
```

Comment:
If you confirm that this adds no additional transitive dependencies then let's just make it a normal dependency (and remove the instruction to install it manually in the readme)